Makefile

```diff
@@ -30,13 +30,16 @@ default: dev
 bin: prep
 	@CGO_ENABLED=$(CGO_ENABLED) BUILD_TAGS='$(BUILD_TAGS) ui' sh -c "'$(CURDIR)/scripts/build.sh'"
 
+testonly: 
```

Comment:
Should this be a make target? calling make with this target seems to be doing nothing. What do you think changing it to a variable?

Reply:
I'm guessing it's here as a dependency target, not one to call directly, as a way of making sure the appropriate targets further down in the file get the `testonly` build tag applied.